app/src/main/java/com/hedvig/app/feature/offer/ui/checkout/CheckoutViewModel.kt

```diff
@@ -58,7 +58,8 @@ class CheckoutViewModel(
                     bundleName = response.data.quoteBundle.displayName,
                     netAmount = response.data.netMonthlyCost(),
                     grossAmount = response.data.grossMonthlyCost(),
-                    market = marketManager.market
+                    market = marketManager.market,
+                    email = response.data.quoteBundle.quotes.firstOrNull { it.email != null }?.email
```

Comment:
I like the built-in function for this in the kotlin standard library, firstNotNullOf, which does exactly what you're doing here.
```suggestion
                    email = response.data.quoteBundle.quotes.firstNotNullOfOrNull(OfferQuery.Quote::email)
```
Or this is the same thing: `email = response.data.quoteBundle.quotes.firstNotNullOfOrNull { it.email }`

Reply:
Brilliant